gitenberg/config.py

```diff
@@ -78,6 +78,9 @@ def parse(self):
         global data
         data = yaml.load(self.read())
         data = {} if data is None else data
+        for key, value in os.environ.items():
+          if key.startswith('gitberg_'):
+            data[key[len('gitberg_'):]] = value
```

Comment:
Do you think this is clearer than `key[8:]` ?

Reply:
I find it much easier than counting characters to verify they match.